newm-chain-db/src/main/kotlin/io/newm/chain/database/repository/LedgerRepository.kt

```diff
@@ -51,7 +51,7 @@ interface LedgerRepository {
 
     fun queryUtxoHavingAddress(address: String, hash: String, ix: Int): Utxo?
 
-    fun queryAddressTxLogsAfter(address: String, afterTxId: String?): List<ByteArray>
+    fun queryAddressTxLogsAfter(address: String, afterTxId: String?, limit: Int, offset: Long): List<ByteArray>
```

Comment:
Should we add default values for limit and offset or make them nullable?

Reply:
This is only called internally and we have well-defined values to use each time it is called.